### DISCUSSION TITLE
Admin panel for an existing project ?

### DISCUSSION THREAD

**Discussion by lironsam** [filamentphp/filament#353] [Help]
Hi,

I just came across Filament and it looks awesome ! Thank you guys.
I have already a project running without admin panel (just doing some auth and role checking directly on the code). I would like to know if it's possible to generate a Filament Admin for my project and directly interact with my data ? Or at least generate CRUD for some of my tables that are already exists on my DB ?
It would be great.

Thank a lot

**Comment by danharrin** [comment]
Yup! You can install Filament into any existing project that's running Laravel 8.
